Makefile

```diff
@@ -0,0 +1,48 @@
+GO_VET_CMD      = go tool vet --all -shadow
+GO_DIRS         = $(shell ls -d */ | \grep -v vendor)
+GO_PKGS         = $(shell go list ./... | \grep -v vendor)
+GO_SRCS         = $(shell find . -type f -name '*.go')
+
+# assets removes the existing bindata.go and regenerates it.
+.PHONY          : assets
+assets          : clean resources/bindata.go
```

Comment:
nitpick... I suggest you start a new line for following command, like:
assets: clean resources/bindata.go

Reply:
You mean new line between PHONY and assets?

Reply:
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
No, I mean start a new line for `clean resources/bindata.go`

Reply:
I am sorry, I don't understand what you mean. Could you provide an example? You mean a new line between `assets:` and `clean resources/bindata.go`?

Reply:
You can find example in the link I attached. Like below:
.PHONY: clean
clean:
        rm *.o temp

Reply:
I really don't understand sorry, the code is already like this.

```
.PHONY: clean
clean:
        rm -f resources/bindata.go
```

But as you requested to remove the bindata generation, I"ll just remove the target altogether.